src/assets/samples.js

```diff
@@ -7,6 +9,9 @@ export const samples = [
         FileName: 'loco.dyn',
         FilePath:
           'D:\\ProgramData\\Autodesk\\RVT 2025\\Dynamo\\non-samples\\Loco_file.dyn',
+        Thumbnail: img,
+        Description: "Test description",
+        DateModified: '1/9/2024 6:24:35 PM',
```

Comment:
why is datemodified hardcoded here?

Reply:
This is just a resource used during development. It will be loaded if no data is provided from the backend. Otherwise, not shown.

Reply:
So can we just keep it empty then?

Reply:
confirmed with Deyan, this change should be fine.

Reply:
It's just for testing purposes, it's safe to keep and handy when working with the app.